charts/stable/grocy/values.yaml

```diff
@@ -6,27 +6,39 @@
 #
 
 image:
+  # -- image repository
   repository: linuxserver/grocy
-  tag: version-v2.7.1
+  # -- image tag
+  tag: version-v3.0.1
+  # -- image pull policy
   pullPolicy: IfNotPresent
 
-strategy:
-  type: Recreate
-
-env: {}
-  # TZ:
-  # PUID:
-  # PGID:
+# -- environment variables. See more environment variables in the [${CHARTNAME} documentation](https://${CHARTNAME}.org/docs).
```

Comment:
Looks like there was a small copy/paste issue here :)

Reply:
I think this is due to tiredness.